src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/ReplicationProtectedItem/StartAzureRmRecoveryServicesAsrTestFailoverJob.cs

```diff
@@ -307,6 +319,19 @@ private void StartRPITestFailover()
                     string.Format(
                         Resources.UnsupportedReplicationProviderForTestFailover,
                         this.ReplicationProtectedItem.ReplicationProvider));
+            }else if (0 == string.Compare(
```

Comment:
Please do a proper string comparison (apply everywhere)

Reply:
Taking in new code.